python-client/trustedanalytics/core/graph.py

```diff
@@ -84,15 +84,16 @@ def __repr__(self):
 
     @api
     @property
-    @returns(data_type=str, description="Status of the graph")
+    @returns(data_type=str, description="""Status of the graph.
+One of three statuses: ``Active``, ``Deleted``, ``Deleted_Final``\:
+
+|   Active:   available for use
```

Comment:
This is stale --what you've deleted is most correct and up to date.  The states are "Active", "Dropped", and "Finalized"